src/js/help/renderGallery.js

```diff
@@ -1,42 +1,88 @@
-'use strict';
-//DOM
-const gallery = document.querySelector('.gallery');
-
-const createItem = item => {
-  const { comments, downloads, likes, largeImageURL, tags, previewURL, views } =
-    item;
-  let itemMarkup = `<a class="gallery__item" href="${largeImageURL}" toClear>
-  <img class="gallery__image" src="${previewURL}" alt="${tags}" loading="lazy"/>
-  <div class="gallery__image-info">
-    <div class="gallery__image-info-item">
-        <p><b>Likes</b></p>
-        <p class="gallery__image-info-display">${likes}</p>
-    </div>
-        <div class="gallery__image-info-item">
-        <p><b>Views</b></p>
-        <p class="gallery__image-info-display">${views}</p>
-    </div>
-    <div class="gallery__image-info-item">
-        <p><b>Comments</b></p>
-        <p class="gallery__image-info-display">${comments}</p>
+import axios from 'axios';
+
+const movieBox = document.querySelector('.box');
+const loader = document.querySelector('.loader');
+const apiKey = '4f9b3bc6cd1b3d6e0d830ad9a5ccfefd';
+
+
+let movieID = [];
+let movieDetails = [];
+
+async function fetchingMovies(page = 1) {
+  const media_type = 'movie';
+  const time_window = 'week';
+  try {
+    const response = await axios.get(
+      `https://api.themoviedb.org/3/trending/${media_type}/${time_window}?api_key=${apiKey}&page=${page}`
+    );
+    // console.log(response.data);
+    return response.data;
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+function renderMovieList(moviesData) {
+  moviesData.results.forEach(movie => {
+    movieID.push(movie.id);
+  });
+  return movieID;
+}
+
+async function fetchingMovieDetails() {
+  for (let id of movieID) {
+    await fetch(`https://api.themoviedb.org/3/movie/${id}?api_key=${apiKey}`)
+      .then(response => {
+        if (!response.ok) {
+          throw new Error(response.status);
+        }
+        return response.json();
+      })
+      .then(data => movieDetails.push(data));
+  }
+  // console.log(movieDetails);
+}
+
+async function updatingMovieHTML() {
```

Comment:
to jest główna część która powinna powstać i być eksportowana aby inni mogli z niej skożystać :)